_styles/header.scss

```diff
@@ -60,13 +60,16 @@ header a {
 
 // main title
 .title > *:first-child {
-  font-size: var(--large);
+  color:white; 
+  font-size: var(--xxxl);
+  
 }
 
 // subtitle
 .title > *:last-child {
   color: var(--gray);
   font-weight: var(--thin);
+  opacity: 0;
```

Comment:
Oh I see! If you make the subtitle empty text in the config file, can you remove this?

Reply:
I couldn't keep the subtitle section empty in config.yaml since it made the main title smaller. I tried removing the subtitle section from the header file, but that did not change.